### PR TITLE
All the hoods

### PR DESCRIPTION
**Pull request by k-nut**
This adds a marker per hood to the map. 

TODO
- [x] Add proper icon
- [x] Call `init()` again or find another way to load the data for the new location 

What do you think @jochenklar ?

### CODE REVIEW

**Comment by k-nut** [comment]
We are still awaiting a proper icon from Julian to show for the individual cities in the zoomed out state.

**Comment by k-nut** [comment]
@jochenklar ready for review now.

**Comment by jochenklar** [comment]
The hood selector was shown along all the other Berlin markers, when not providing any hood. I disabled the default hood `berlin` and added a default center and zoom to `init()` (all of Germany). Also I added a `title` field in order to have a readable link. Plz check and merge :smiley:

File: app.js [hunk 1]
```diff
@@ -23,90 +25,103 @@
...
         marburg: {
+            title: 'Marburg',
             airtable_url: 'https://api.airtable.com/v0/appt1VwCeE69OXvSa/',
             airtable_key: 'keyl5v0iA9uirvIAH',
             border: 'border/marburg.json',
             center: [50.814666650503554, 8.773441314697266,],
             zoom: 15
         },
         muelheim: {
+            title: 'Mühlheim',
```
Comment (k-nut) [inline app.js]:
```suggestion
            title: 'Mülheim'
```

**Pull request merged**